scripts/adminrouter-agent-setup.ps1

```diff
@@ -14,142 +15,54 @@ Import-Module $utils
-    Add-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST/adminrouter'  -filter "system.webServer/rewrite/outboundRules" -name "." -value @{name='outbound rewrite';preCondition='IsHTML'}
-    Set-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST/adminrouter'  -filter "system.webServer/rewrite/outboundRules/rule[@name='outbound rewrite']/match" -name "filterByTags" -value "A"
-    Set-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST/adminrouter'  -filter "system.webServer/rewrite/outboundRules/rule[@name='outbound rewrite']/match" -name "pattern" -value "^(.*)"
-    Add-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST/adminrouter'  -filter "system.webServer/rewrite/outboundRules/rule[@name='outbound rewrite']/conditions" -name "." -value @{input='{URL}';pattern='^/(system/health/v1|system/v1/metrics)/.\*'}
-    Set-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST/adminrouter'  -filter "system.webServer/rewrite/outboundRules/rule[@name='outbound rewrite']/action" -name "type" -value "Rewrite"
-    Set-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST/adminrouter'  -filter "system.webServer/rewrite/outboundRules/rule[@name='outbound rewrite']/action" -name "value" -value "/{C:1}/{R:1}"
-    Add-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST/adminrouter'  -filter "system.webServer/rewrite/outboundRules" -name "." -value @{name='Rewrite location header';preCondition='IsRedirection'}
-    Set-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST/adminrouter'  -filter "system.webServer/rewrite/outboundRules/rule[@name='Rewrite location header']/match" -name "serverVariable" -value "RESPONSE_Location"
-    Set-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST/adminrouter'  -filter "system.webServer/rewrite/outboundRules/rule[@name='Rewrite location header']/match" -name "pattern" -value "http://[^/]+/(.*)"
-    Add-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST/adminrouter'  -filter "system.webServer/rewrite/outboundRules/rule[@name='Rewrite location header']/conditions" -name "." -value @{input='{ORIGINAL_HOST}';pattern='.+'}
-    Add-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST/adminrouter'  -filter "system.webServer/rewrite/outboundRules/rule[@name='Rewrite location header']/conditions" -name "." -value @{input='{URL}';pattern='^(system/health/v1|system/v1/metrics)/.\*'}
-    Set-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST/adminrouter'  -filter "system.webServer/rewrite/outboundRules/rule[@name='Rewrite location header']/action" -name "type" -value "Rewrite"
-    Set-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST/adminrouter'  -filter "system.webServer/rewrite/outboundRules/rule[@name='Rewrite location header']/action" -name "value" -value "http://{ORIGINAL_URL}/{C:1}/{R:1}"
+function New-AdminRouterAgent {
+   $logFile = Join-Path $ADMINROUTER_LOG_DIR "adminrouter-agent.log"
```

Comment:
Small suggestion to this line. Please indent it 4 spaces instead of 3 spaces. In order to keep the indentation consistent.

Reply:
Oops. I will fix it.